packages/blazor-workspace/Examples/Demo.Shared/Pages/ComponentsDemo.razor

```diff
@@ -171,19 +171,35 @@
         <div class="sub-container">
             <div class="container-label">Select</div>
             <NimbleSelect Appearance="DropdownAppearance.Underline" FilterMode="FilterMode.Standard" Clearable="true">
-                <NimbleListOption>Option 1</NimbleListOption>
-                <NimbleListOption>Option 2</NimbleListOption>
-                <NimbleListOption>Option 3</NimbleListOption>
+                <NimbleListOptionGroup Label="Group 1">
+                    <NimbleListOption>Option 1</NimbleListOption>
+                    <NimbleListOption>Option 2</NimbleListOption>
+                </NimbleListOptionGroup>
+                <NimbleListOptionGroup Label="Group 2">
+                    <NimbleListOption>Option 3</NimbleListOption>
+                    <NimbleListOption>Option 4</NimbleListOption>
+                </NimbleListOptionGroup>
             </NimbleSelect>
+            <NimbleListOptionGroup></NimbleListOptionGroup>
```

Comment:
Can you make sure this aligns with Blazor? Two things:
- Don't think we need this empty listoptiongroup that is outside of any select
- Blazor does not seem to show placeholders, can't remember, are we able to use placeholders in blazor?

Reply:
That first one was just an "oopsy". For the second, we haven't enabled this because of  [this issue](https://github.com/ni/nimble/issues/1952).